dataRetriever/dataPool/proofsCache/proofsCache.go

```diff
@@ -14,15 +14,15 @@ type proofNonceMapping struct {
 
 type proofsCache struct {
 	mutProofsCache       sync.RWMutex
-	proofsByNonceBuckets []*proofNonceBucket
-	bucketSize           int
+	proofsByNonceBuckets sync.Map
```

Comment:
proofNonceMapping struct not used anymore

Reply:
:+1: delete it